tests/test_md.py

```diff
@@ -24,6 +27,7 @@
 
 relaxer = StructOptimizer()
 structure = Structure.from_file(f"{ROOT}/examples/mp-18767-LiMnO2.cif")
+structure_2 = Structure.from_file(f"{ROOT}/examples/mp-1175469-Li9Co7O16.cif")
```

Comment:
Why is there a need to include LCO for Nose-Hoover test?

Reply:
> Why is there a need to include LCO for Nose-Hoover test?

Because the cell matrix of LiMnO2 is diagonal.

Reply:
This is makes sense. Another way is to use rotation and non-diagonal supercell expansion to create a non-diagonal LiMnO2 cell